app/reports/hbx_reports/edi_enrollment_termination_report.rb

```diff
@@ -29,7 +30,14 @@ def migrate
         processed_count = 0
 
         time_stamp = Time.now.utc.strftime("%Y%m%d_%H%M%S")
-        file_name = File.expand_path("#{Rails.root}/public/edi_enrollment_termination_report_#{time_stamp}.csv")
+        file_name = if individual_market_is_enabled?
+                      File.expand_path("#{Rails.root}/public/edi_enrollment_termination_report_#{time_stamp}.csv")
+                    else
+                      # For MA stakeholders requested a specific file format
+                      file_format = fetch_CCA_required_file_format
```

Comment:
I think you've done a great job but yea I agree with you, let's DRY up these lines by using a helper method.